cpp/src/arrow/c/bridge_test.cc

```diff
@@ -4330,6 +4334,117 @@ TEST_F(TestDeviceArrayRoundtrip, Struct) {
   TestWithJSON(mm, type, R"([[4, null], null, [5, "foo"]])");
 }
 
+#ifdef ARROW_CUDA
```

Comment:
To avoid CUDA-specific code in non-CUDA directories, perhaps move this test somewhere in `src/arrow/gpu`?

Reply:
Yes, will move to `cuda_test.cc`